src/en/dynasty/src/eu/kanade/tachiyomi/extension/en/dynasty/Dynasty.kt

```diff
@@ -0,0 +1,698 @@
+                    append(type, ":\n")
+                    values.forEach { append("• ", it.second, "\n") }
+                    append("\n")
+                }
+                append("Released: ", data.releasedOn)
+            }.trim()
+            genre = tags.joinToString()
+            thumbnail_url = buildCoverUrl(data.pages.first().url)
+            status = SManga.COMPLETED
+            update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
+        }
+    }
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        return if (manga.url.contains("/chapters/")) {
+            Observable.just(
+                listOf(
+                    SChapter.create().apply {
+                        url = manga.url
+                        name = "Chapter"
+                        date_upload = dateFormat.tryParse(
+                            manga.description
+                                ?.substringAfter("Released:", ""),
+                        )
+                    },
+                ),
+            )
+        } else {
+            super.fetchChapterList(manga)
+        }
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return mangaDetailsRequest(manga)
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val data = response.parseAs<MangaResponse>()
+        val chapters = data.taggings.toMutableList()
+
+        var page = 2
+        val limit = preferences.chapterFetchLimit
+
+        while (page <= data.totalPages && page <= limit) {
+            val url = response.request.url.newBuilder()
+                .addQueryParameter("page", page.toString())
+                .build()
+
+            chapters += client.newCall(GET(url, headers)).execute()
+                .parseAs<MangaResponse>().taggings
+            page += 1
+        }
+
+        var header: String? = null
+
+        val chapterList = mutableListOf<SChapter>()
+
+        chapters.forEach { item ->
+            if (item is MangaChapterHeader) {
+                header = item.header
+                return@forEach
+            }
+
+            with(item as MangaChapter) {
+                var chapterName = header?.let { "$it $title" } ?: title
+                if (data.type != "Series") {
+                    chapterName += tags.filter { it.type == "Author" }
+                        .joinToString(prefix = " by ", separator = " and ") { it.name }
+                }
+                SChapter.create().apply {
+                    url = "/chapters/$permalink"
+                    name = chapterName
+                    scanlator = tags.filter { it.type == "Scanlator" }.joinToString { it.name }
+                    date_upload = dateFormat.tryParse(releasedOn)
+                }.also(chapterList::add)
+            }
+        }
+
+        return if (data.type != "Doujin") {
+            chapterList.asReversed()
+        } else {
+            chapterList
+        }
```

Comment:
That's a weird condition, is there some more reliable indicator the list is reversed?

Reply:
only doujins are ordered based on their date uploaded, while everything else is ordered based on number on the site. 

This behavior is same in old code